server/submission/resolvers.test.js

```diff
@@ -278,8 +206,20 @@ describe('Submission', () => {
       const expectedManuscript = lodash.merge(initialManuscript, manuscript)
       expectedManuscript.submissionMeta.stage = 'QA'
       expect(expectedManuscript).toMatchObject(storedManuscript)
+    })
+
+    it('sends Corresponding Author a verification email', async () => {
```

Comment:
Also add a test that it doesn't send an email if the submitter and author emails are equal?

Reply:
`if (ctx.user.email !== newManuscript.submissionMeta.author.email)` always resolves to true, because `ctx.user.email` is undefined. So, currently, an email is always sent.

Checking both of this situations properly will only be possible (or at least meaningful) once we're saving the submitter's details.

Reply:
In the tests you can set up the user to have the appropriate email and therefore not send the mail. I would say that the fact it doesn't work in the app (and therefore can't be end-to-end tested) is all the more reason to fully unit test it.

Reply:
Done! :grin: